src/components/core/PivotTable.tsx

```diff
@@ -331,6 +331,10 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         return this.state.execution ? this.state.execution.executionResponse : null;
     };
 
+    private getAfmFilters = () => {
```

Comment:
I am wondering, whether an empty array instead of `null` wouldn't be better here. Below you are using `getAfmFilters: () => AFM.CompatibilityFilter[] | null;` and also the interface:
```
getAfmFilters: () => AFM.CompatibilityFilter[] | null;
```

Reply:
Good suggestion. The code is cleaner and it makes more sense now 👍